bigtop-packages/src/common/ambari/patch5-refine-AMBARI-25599-for-Hadoop-3.4.diff

```diff
@@ -93,13 +58,13 @@ index a0a11b8e6e..f33479844a 100644
 -    <grafana.tar>https://dl.grafana.com/oss/release/grafana-6.4.2.linux-amd64.tar.gz</grafana.tar>
 -    <phoenix.tar>https://s3.amazonaws.com/dev.hortonworks.com/HDP/centos7/3.x/BUILDS/3.1.4.0-315/tars/phoenix/phoenix-5.0.0.3.1.4.0-315.tar.gz</phoenix.tar>
 -    <phoenix.folder>phoenix-5.0.0.3.1.4.0-315</phoenix.folder>
-+    <hbase.tar>https://archive.apache.org/dist/hbase/2.4.13/hbase-2.4.13-bin.tar.gz</hbase.tar>
++    <hbase.tar>http://bigtop-snapshot.s3.amazonaws.com/centos-7/x86_64/bigtop-stack-binary/hbase-2.4.13-bin.tar.gz</hbase.tar>
 +    <hbase.folder>hbase-2.4.13</hbase.folder>
-+    <hadoop.tar>https://archive.apache.org/dist/hadoop/common/hadoop-3.3.4/hadoop-3.3.4.tar.gz</hadoop.tar>
++    <hadoop.tar>http://bigtop-snapshot.s3.amazonaws.com/centos-7/x86_64/bigtop-stack-binary/hadoop-3.3.4.tar.gz</hadoop.tar>
 +    <hadoop.folder>hadoop-3.3.4</hadoop.folder>
 +    <grafana.folder>grafana-9.1.2</grafana.folder>
 +    <grafana.tar>https://dl.grafana.com/enterprise/release/grafana-enterprise-9.1.2.linux-amd64.tar.gz</grafana.tar>
-+    <phoenix.tar>https://archive.apache.org/dist/phoenix/phoenix-5.1.2/phoenix-hbase-2.4-5.1.2-bin.tar.gz</phoenix.tar>
++    <phoenix.tar>http://bigtop-snapshot.s3.amazonaws.com/centos-7/x86_64/bigtop-stack-binary/phoenix-hbase-2.4-5.1.2-bin.tar.gz</phoenix.tar>
 +    <phoenix.folder>phoenix-hbase-2.4-5.1.2-bin</phoenix.folder>
```

Comment:
While I don't think this is the right direction, I'm going to merge this if you can't proceed the release process of Bigtop 3.2.

Reply:
Thanks for you review, @iwasakims , after some tests on ambari-metrics, we found there are dependency issues when using tarballs provided by their own community